apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -42,6 +56,146 @@ int main(int argc, char **argv)
 	fprintf(stdout, "%15s: Hello, World!! on stdout\n", argv[0]);
 	fprintf(stderr, "%15s: Hello, World!! on stderr\n", argv[0]);
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	volatile uint32_t *address;
+	char ch;
+
+	printf("\nPress R - For Read Test\n");
+	printf("Press W - For Write Test\n");
+
+	ch = getchar();
+	if (ch == 'R' || ch == 'r') {
+		printf("\nPress A for app MPU test\n");
+		printf("Press K for kernel MPU test\n");
+
+		ch = getchar();
+
+		if (ch == 'A' || ch == 'a') {
+			address = APP_START_ADDR;
+
+			uint32_t dest;
+
+			printf("\n************************************************\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+			printf("* Test to verify protection of Micom app space *\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+
+			printf("INFO: Read Micom app space: 0x%x\n", address);
+			sleep(1);
+			dest = *address;
+
+			printf("INFO: Wifi Task successfully read Micom app space\n");
```

Comment:
If this message is showing, test is not working properly.
But the log message seems to run well.
How about changing the log message like below?
`INFO : Wifi Task accesses the invalid address which is in Micom app space`

Reply:
Fixed.